lua/weapons/cfc_weapon_parachute/init.lua

```diff
@@ -494,3 +509,20 @@ function SWEP:KeyPress( ply, key, state )
         self:SetChuteDirection()
     end
 end
+
+function SWEP:UpdateMoveKeys()
+    local owner = self:GetOwner()
+    owner = IsValid( owner ) and owner or self.chuteOwner
+
+    if not IsValid( owner ) or not owner:IsPlayer() then return end
```

Comment:
```suggestion
    if not ( IsValid( owner ) or owner:IsPlayer() ) then return end
```
I think this reads a little bit better, reading the logic here was a bit confusing to me.

Reply:
`not COND or not COND`, etc is the format most often used in CFC code.

Also, the point of doing it this way is to make use of short-circuiting. In the first case, if `owner` is invalid, it instantly skips to the return. In the second case, it'll still try to evaluate `owner:IsPlayer()` which in some cases might yield a nil error, if `owner` is something other than an entity, such as if it was `nil`. It'd use up more time since it checks both values every time, and in some rare cases could result in an error.

Reply:
why not just split the statements up